src/Test_FCS_load_and_transform.jl

```diff
@@ -0,0 +1,198 @@
+# Load and transform
+# build the general workflow to have the data ready
+# for processing with GigaSOM
+
+#=
+using FCSFiles for loading
+as this function is only the basic parsing of the binary
+FCS, we need to see what functionality is missing and
+extend this in the original package
+=#
+
+
+Pkg.add("FCSFiles")
+Pkg.add("DataFrames")
+Pkg.add("CSV")
+Pkg.add("StatsPlots")
+Pkg.add("Statistics")
+
+
+Pkg.add("MultivariateStats")
+
+# cd("/home/ohunewald/work/GigaSOM_data/test_data")
+cd("/home/ohunewald/work/GigaSOM_data/PBMC8_fcs_files")
```

Comment:
I think we shall either fetch these dynamically or put them on ARTENOLIS directly.